yatdlm/todo/models.py

```diff
@@ -250,6 +267,14 @@ def get_categories(self):
 
         return [cat.as_dict() for cat in categories]
 
+    def set_categories(self, categories_ids: T.List[int]) -> None:
```

Comment:
You keep using pre-3.9 syntax and 3.9 syntax interchangeably. Built-in `list` is subscriptable since that version so this could be `list[int]` like you do in other parts of the code (with `dict`).